src/Microsoft.DocAsCode.Build.Engine/PostProcessors/ValidateBookmark.cs

```diff
@@ -97,6 +97,12 @@ protected override void HandleCore(HtmlDocument document, ManifestItem manifestI
                      TargetLineNumber = node.Line
                  }).ToList();
             var anchors = GetNodeAttribute(document, "id").Concat(GetNodeAttribute(document, "name"));
+            if (manifestItem.Metadata.ContainsKey("rawTitle"))
```

Comment:
I see here is `rawTitle`, while you assign the value to `RawTitle`. Have you tested this can work?

Reply:
Tested, it's working.